CHANGELOG.md

```diff
@@ -3,6 +3,103 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.0
+
+_Aug 1, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+### 💵 Our commercial offering is evolving!
```

Comment:
If you choose to put that into a h3 then we should move the sentence above ("We'd like ...") inside the "Highlights" h3 and we should remove one level of indentation for the paragraph about the new commercial offering

Reply:
No strong preference about using a dedicated h3 or not, I think it can make sense

Reply:
Ty, I made the changes from github UI

I think in the MD the H3 makes the changes clearer and adds importance to it

Reply:
Having @joserodolfofreitas on the content of this section would be nice before releasing :ok_hand: